backend/test/mocking.clj

```diff
@@ -1,15 +1,25 @@
 (ns mocking
```

Comment:
Sorry, I don't see myself reviewing this mocking. I don't get *how* it does what it does.

Reply:
`with-redefs` replaces function definitions within its context, and wherever we want it we apply it to all the `defexpect`s in a namespace.

The result is that instead of the real persistence functions being called, those defined here are called.

The other concept is simply that an `atom` is a *persistent* and *mutable* object which can be changed with `reset! atom value`.

If you have any other doubts I can clear them, I think it's important that I'm not the only one who understands this :upside_down_face: